code-challenges.test.js

```diff
@@ -25,35 +38,140 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+describe("divByThree", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+    const object1 = { number: 15 }
+    const object1OutPut = "15 is divisible by three"
+    expect(divByThree(object1.number)).toEqual(object1OutPut)
+    const object2 = { number: 0 }
+    const object2OutPut = "0 is divisible by three"
+    expect(divByThree(object2.number)).toEqual(object2OutPut)
+    const object3 = { number: -7 }
+    const object3OutPut = "-7 is not divisible by three"
+    expect(divByThree(object3.number)).toEqual(object3OutPut)
+    })
+})
+
+// ReferenceError: divByThree is not defined
+
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
+/*
+Create a function named divByThree that has only one parameter that takes an object as an argument
+    Use a conditional statement to determine 
+    If the parameter is divisible by three by using the modulus to strictly equal zero
+    return the output that the object is divisible by 3
+    if the object is not divisible by 3
+        return the object is not divisible by 3
+    For an edge case
+    if the object is not a number
+        return the object is not a number
+*/
+
+const divByThree = (object) => {
+    if (object % 3 === 0) {
+        return `${object} is divisible by three`
+    } else if (object % 3 !== 0) {
+        return `${object} is not divisible by three`
+    } else {
+        return `${object} is not a number`
+    }
+}
+
+// Originally I was getting a syntax error and that the input of the object was undefined.
+//In order to get the correct input I needed, I used dot notation in order to access the value from the key: number. 
```

Comment:
As mentioned above, that process should be handled inside of your function and not inside of your test.  Otherwise, this looks excellent!